src/test/java/org/minions/devfund/david/Movie/MovieDatabaseTest.java

```diff
@@ -0,0 +1,141 @@
+package org.minions.devfund.david.Movie;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Movie database tests class.
+ */
+public class MovieDatabaseTest {
+    private static final String NEW_MOVIE = "NewMovie";
+    private static final String OLD_MOVIE = "OldMovie";
+    private static final String ACTOR_1 = "Actor1";
+    private static final String ACTOR_2 = "Actor2";
+    private MovieDatabase movieDatabase;
+
+    /**
+     * Setup movie database class.
+     */
+    @Before
+    public void setupMovieDatabase() {
+        movieDatabase = new MovieDatabase();
+    }
+
+    /**
+     * Test get best actor.
+     */
+    @Test
+    public void testGetBestActor() {
+        final double newMovieRating = 8.0;
+        movieDatabase.addMovie(NEW_MOVIE, new String[]{ACTOR_1, ACTOR_2});
+        movieDatabase.addRating(NEW_MOVIE, newMovieRating);
+
+
+        final double oldMovieRating = 10.0;
+        movieDatabase.addRating(OLD_MOVIE, oldMovieRating);
+        movieDatabase.addMovie(OLD_MOVIE, new String[]{ACTOR_1});
+
+        assertEquals(ACTOR_1, movieDatabase.getBestActor());
+    }
+
+    /**
+     * Test get best movie.
+     */
+    @Test
+    public void testGetBestMovie() {
+        final double newMovieRating = 9.0;
+        movieDatabase.addMovie(NEW_MOVIE, new String[]{ACTOR_2});
+        movieDatabase.addRating(NEW_MOVIE, newMovieRating);
+
+
+        final double oldMovieRating = 10.5;
+        movieDatabase.addMovie(OLD_MOVIE, new String[]{ACTOR_1});
+        movieDatabase.addRating(OLD_MOVIE, oldMovieRating);
+
+        assertEquals(OLD_MOVIE, movieDatabase.getBestMovie());
+    }
+
+    /**
+     * Test add movie with actor that exist.
+     */
+    @Test
+    public void testAddMovieActorExist() {
+        movieDatabase.addMovie(NEW_MOVIE, new String[]{ACTOR_1, ACTOR_2});
+        final int expectedActorSize = 2;
+
+        assertEquals(expectedActorSize, movieDatabase.getActorList().size());
+
+        String actor3 = "Actor1";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use already-defined constant 'ACTOR_1' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)